### PR TITLE
Added Dodo code translations

### PR DESCRIPTION
**Pull request by MrOgeid**
Added some missing strings too. Minor text fixes for better understandable words.

### CODE REVIEW

**Comment by MrOgeid** [comment]
I noticed a wrong translation that I can find a way to correct. Probably because Italian has different words for Orange the color and Orange the fruit. Italian has a big and sometime strange lexicon.🙄😅 

I know it's a very secondary problem but maybe there's a way to have two different translation for the same word based of where the word is used in the app?

**Comment by TheVaan** [comment]
Well, this is not possible as we are translating stings that came directly from API. So the translation key can't be modified by us. Sorry :/

**Comment by TheVaan** [comment]
LGTM, thank you!

File: ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -1099,19 +1040,30 @@ Puoi annullare il tuo abbonamento in ogni momento dalle impostazioni del tuo acc
...
+"Please fill all the fields" = "Per favore compila tutti i campi";
+"Dodo code" = "Codice Dodo";
+"Dodo codes" = "Codici Dodo";
+"This Dodo code is invalid" = "Questo Codice Dodo non è valido";
+"Island description" = "Descrizione dell'isola";
+"Native fruit" = "Frutta autoctona";
+"Add your Dodo code" = "Aggiungi il tuo Codice Dodo";
+"Hemisphere: %@" = "Emisfero: %@";
+"Delete" = "Elimina";
+"Report (%lld)" = "Segnalare (%lld)";
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/it.lproj/Localizable.strings:1061]:
```suggestion
"Report" = "Segnalare";
```

String has been renamed.

**Comment by MrOgeid** [comment]
Thank you @TheVaan for changing the renamed string. I missed it!
For the "orange" problem I understood now how it works. I'll try to bypass the problem with a unique translation for both words. It exist in Italian but it's a rather "old way" word.

**Pull request merged**